src/DependencyInjection/DI.Specification.Tests/src/Fakes/FakeDisposableCallbackOuterService.cs

```diff
@@ -13,10 +14,10 @@ public class FakeDisposableCallbackOuterService : FakeDisposableCallbackService,
             FakeDisposeCallback callback) : base(callback)
         {
             SingleService = singleService;
-            MultipleServices = multipleServices;
```

Comment:
This can be null?

Reply:
https://github.com/aspnet/Extensions/pull/1504/checks?check_run_id=107248909